src/app/copy/block-copy/example/block-copy-callback-example.component.ts

```diff
@@ -0,0 +1,46 @@
+import {
+  Component,
+  OnInit,
+  ViewEncapsulation
+} from '@angular/core';
+
+import { BlockCopyConfig } from '../block-copy-config';
+
+import { Notification } from '../../../notification/notification';
+import { NotificationService } from '../../../notification/notification-service/notification.service';
+import { NotificationType } from '../../../notification/notification-type';
+
+@Component({
+  encapsulation: ViewEncapsulation.None,
+  selector: 'block-copy-callback-example',
+  templateUrl: './block-copy-callback-example.component.html'
+})
+export class BlockCopyCallbackExampleComponent implements OnInit {
```

Comment:
Can we rename this example as "notification" Vs "callback"?

It was a little confusing to see the word "callback" in the example. Callbacks were something heavily used in Angular Patternfly. Instead of invoking user provided functions, we've been using events in patternfly-ng.

Considering the example is showing a notification, that may be more appropriate?